packages/parse-argv/parse-argv.0.3.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+maintainer: "Mindy Preston <mindy.preston@docker.com>"
+authors: ["Jon Ludlam" "Magnus Skjegstad" "Mindy Preston"]
+tags: "org:mirage"
+homepage: "https://github.com/mirage/parse-argv"
+doc: "https://mirage.github.io/parse-argv/"
+bug-reports: "https://github.com/mirage/parse-argv/issues"
```

Comment:
Can you add a license?

Reply:
done in [0a30e02](https://github.com/ocaml/opam-repository/pull/23809/commits/0a30e02a5c5106d969d5c5c028c48fcc214ee92f)